### DISCUSSION TITLE
[bug report] Cannot respond to reviewer comments on one PR

### DISCUSSION THREAD

**Discussion by travisoneill** [community/community#6893] [Pull Requests]
Reply panel for PR comments is not showing up on just one PR.  I am able to resolve and unresolve conversations and comment on the PR as a whole, but just not reply to individual comments.  All behavior is normal on other PR's in the same repo.  The issue has persisted through logging out and back in and browser restart.

<img width="959" alt="Screen Shot 2021-10-28 at 11 50 19" src="https://user-images.githubusercontent.com/17502258/139235772-c35a6cba-b5b3-4a4b-92ea-a9516bc50e3f.png">

 
Comment panel at bottom of PR works fine
<img width="998" alt="Screen Shot 2021-10-28 at 11 50 25" src="https://user-images.githubusercontent.com/17502258/139235819-cc207da6-530d-4c8f-b96e-92b744607426.png">

Normal reply panel renders in other PR
<img width="878" alt="Screen Shot 2021-10-28 at 11 50 45" src="https://user-images.githubusercontent.com/17502258/139235986-cee6a429-5e5d-462d-ac18-37aaba6da47f.png">

**Comment by micaelbergeron** [comment]
I had the same issue, and it ended up being a stale pending review I had (Files tab).

For one, maybe it would make sense not to prevent individual comment replies even when you have a PR review pending, or at the very least add a message explaining what is going on.

**Reply by fblampe** [reply]
I'm wondering why responding to previous comments would be disallowed at all... I find myself responding to changes made based on suggestions quite often during subsequent reviews, and it would make the most sense to continue the discussion in the same thread instead of having to resolve the old one and start a new one.

**Reply by StanleySathler** [reply]
Agreed!

I had to Google it to figure out these replies are disabled while you have a pending review. A message would be very helpful in this case.

**Comment by behtin** [comment]
I had a same issue 😣

**Comment by CecilePalant** [comment]
I have the same issue and was able to respond via the pull-request tab in IntelliJ.

**Comment by LukeBrandon** [comment]
I am experiencing this same issue.

**Comment by AlexisWilke** [comment]
Looking at the **Files** tab, I can find that comment and reply there.

I think that the difference is that in one case it is part of a review and in the other case (when we can't reply) the person clicked the "Add single comment". However, that "single comment", I think, should be added to that list of replies (which it isn't...) as if the person was working on a review...

**Comment by pond** [comment]
Basic feature still broken in late 2023; can't reply to some comments, can reply to others, no apparent pattern; just buggy.

**Comment by mackenzie-grimes-noaa** [comment]
> Looking at the **Files** tab, I can find that comment and reply there.
> 
> I think that the difference is that in one case it is part of a review and in the other case (when we can't reply) the person clicked the "Add single comment".

For anyone coming here from another search engine: if you don't see a "Write a reply" box rendering under a comment, look for the comment in the `Files changed` tab of the PR instead.

It seems that if a reviewer submits a comment using the `Start a review` button, GitHub renders a reply text field in the `Files changed` and `Conversation` tabs. If they submit with the `Add single comment` button, the comment only shows a reply field in the `Files changed` tab.

[GitHub docs](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#about-pull-request-comments)

![GitHub Reviewer comment box](https://github.com/community/community/assets/136493179/d92540b1-96ab-4457-8d27-827353378b63)

**Comment by guy-premont-cml** [comment]
It is still there, very annoying. Some comments can be replied to, but not others.

**Comment by ggray-cb** [comment]
I have the same issue in 2025. And, as a special bonus, I can see comments on th PR overview page but not reply to them. But unlike what previous commenter mentioned, if I go into the Files Changed tab, I can't see the comment at all so I can't comment on them.  I guess because I updated the file in the PR? Really, really annoying and makes it hard to have an ongoing discussion about changes.

**Comment by martin-smid-brightpick** [comment]
I find this workflow over engineered, nonintuitive and i don't even see any reason for it.
